launch/load_multiple_tif_launch.xml

```diff
@@ -0,0 +1,23 @@
+<launch>
+    <arg name="rviz" default="true"/>
+    <arg name="location" default="sargans"/>
+    <node pkg="tf2_ros" exec="static_transform_publisher" name="world_map" args="--frame-id world --child-frame-id map"/>
+
+    <node pkg="grid_map_geo" exec="test_tif_loader" name="first_tif_loader" output="screen">
+        <param name="frame_id" value="map"/>
+        <param name="tif_path" value="$(find-pkg-share grid_map_geo)/resources/sargans.tif"/>
```

Comment:
Do you want to instead want me to add a `package.xml` and `CMakeLists.txt` to the `terrain-models` repo so we don't need to symlink across them?

Reply:
Maybe, but the data can not be shared by the public, so we should eventually remove references to the terrain-models models in general

Reply:
Yea, I guess if there is never a way to open source that data, this is good enough.